src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1,1403 +1,1407 @@
+  </data>
+  <data name="GeneralPage_CheckForUpdates.Content" xml:space="preserve">
+    <value>Check for updates</value>
+  </data>
+  <data name="GeneralPage_UpdateNow.Content" xml:space="preserve">
+    <value>Update now</value>
+  </data>
+  <data name="GeneralPage_PrivacyStatement_URL.Label" xml:space="preserve">
+    <value>Privacy statement</value>
+  </data>
+  <data name="GeneralPage_ReportAbug.Label" xml:space="preserve">
+    <value>Report a bug</value>
+    <comment>Report an issue inside powertoys</comment>
+  </data>
+  <data name="GeneralPage_RequestAFeature_URL.Label" xml:space="preserve">
+    <value>Request a feature</value>
+    <comment>Tell our team what we should build</comment>
+  </data>
+  <data name="GeneralPage_RestartAsAdmin_Button.Content" xml:space="preserve">
+    <value>Restart as administrator</value>
+    <comment>running PowerToys as a higher level user, account is typically referred to as an admin / administrator</comment>
+  </data>
+  <data name="GeneralPage_ToggleSwitch_RunAtStartUp.Header" xml:space="preserve">
+    <value>Run at startup</value>
+  </data>
+  <data name="About_PowerRename.ModuleDescription" xml:space="preserve">
+    <value>A Windows Shell extension for more advanced bulk renaming using search and replace or regular expressions.</value>
+  </data>
+  <data name="PowerRename_ShellIntegration.Text" xml:space="preserve">
+    <value>Shell integration</value>
+    <comment>This refers to directly integrating in with Windows</comment>
+  </data>
+  <data name="PowerRename_Toggle_Enable.Header" xml:space="preserve">
+    <value>Enable PowerRename</value>
+    <comment>do not loc the Product name.  Do you want this feature on / off</comment>
+  </data>
+  <data name="RadioButtons_Name_Theme.Text" xml:space="preserve">
+    <value>Settings theme</value>
+  </data>
+  <data name="PowerRename_Toggle_EnableOnContextMenu.Header" xml:space="preserve">
+    <value>Show icon on context menu</value>
+  </data>
+  <data name="PowerRename_Toggle_EnableOnExtendedContextMenu.Header" xml:space="preserve">
+    <value>Appear only in extended context menu (Shift + Right-click)</value>
+  </data>
+  <data name="PowerRename_Toggle_MaxDispListNum.Header" xml:space="preserve">
+    <value>Maximum number of items</value>
+  </data>
+  <data name="PowerRename_Toggle_RestoreFlagsOnLaunch.Content" xml:space="preserve">
+    <value>Show values from last use</value>
+  </data>
+  <data name="FileExplorerPreview_ToggleSwitch_Preview_MD.Header" xml:space="preserve">
+    <value>Enable Markdown (.md) preview</value>
+    <comment>Do not loc "Markdown".  Do you want this feature on / off</comment>
+  </data>
+  <data name="FileExplorerPreview_ToggleSwitch_Preview_SVG.Header" xml:space="preserve">
+    <value>Enable SVG (.svg) preview</value>
+    <comment>Do you want this feature on / off</comment>
+  </data>
+  <data name="FileExplorerPreview_ToggleSwitch_Preview_PDF.Header" xml:space="preserve">

```

Comment:
This file has been rewritten to have different line endings than the one in the repository; would you mind re-saving it via visual studio? That would likely help with the merge conflicts as well.

(Rewriting the line endings is a huge source of merge conflicts, since... git can apparently not figure it out here in the year 2021 :smile:)